File: frontend/ksweb-app/src/app/pages/usuarios/usuarios.models.ts
export interface UsuarioListItem {
  usrCodigo: number | null;
  userId: number;
  usrNome: string;
  usrEmail: string;
  usrNivel: string;
  queueId: number | null;
  setor: string | null;
}

export interface UsuarioFiltro {
  page: number;
  pageSize: number;
  termo: string;
}

File: backend/KSWeb.Api/Models/UsuariosModels.cs
namespace KSWeb.Api.Models;

public sealed class UsuarioListItem
{
    public int? UsrCodigo { get; set; }
    public long UserId { get; set; }
    public string UsrNome { get; set; } = string.Empty;
    public string UsrEmail { get; set; } = string.Empty;
    public string UsrNivel { get; set; } = string.Empty;
    public long? QueueId { get; set; }
    public string Setor { get; set; } = string.Empty;
}